addons/wardrobe/config.cpp

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        author = ECSTRING(common,ACETeam);
+        authors[] = {"OverlordZorn [CVO]"};
+        url = ECSTRING(main,URL);
+
+        VERSION_CONFIG;
+        requiredVersion = REQUIRED_VERSION;
+
+        requiredAddons[] = {
+            "ace_interact_menu",
+            "ace_goggles"
+        };
+        skipWhenMissingDependencies = 1;
+
+        units[] = {};
+        weapons[] = {};
+    };
+};
+
```

Comment:
Change entry order so it matches https://github.com/acemod/ACE3/blob/master/addons/compat_cup_vehicles/config.cpp